### ISSUE TITLE
[Bug]: Unexpected behaviour: on start, the search bar is unfocused and ctrl-F does not focus it

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [FreeTubeApp/FreeTube#2308]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the [issue tracker](https://github.com/FreeTubeApp/FreeTube/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.

### Describe the bug

Upon starting the program, the search bar is unfocused and ctrl-F does not focus it. 

I have encountered these behaviours (or lack of behaviours) on both the platforms on which I use FreeTube. Those platforms are Linux and Windows. (I give details below on those systems of mine.)

### Expected Behavior

Ideally, upon starting the program the search bar would be focused. At the least ctrl-F should focus it.

### Issue Labels

accessibility issue, usability issue

### FreeTube Version

v0.16.0 Beta

### Operating System Version

Linux Mint 20.2 Cinnamon. Also, Windows 10 Pro - 21H2 (2009: 19044).

### Installation Method (If applicable)

other

### Primary API used

Local API

### Additional Information

On Windows the version of Freetube that I use is the x64 installable version.
On Linux the version of Freetube that I use is the x64 installable version, i.e., the .deb.

Thank you for your software.

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Cant reproduce, does this also happen on the latest nightly build 

https://github.com/FreeTubeApp/FreeTube/actions/runs/2480241086

**Comment by LinuxOnTheDesktop** [comment]
Yes.

More fully: just now I tried the nightly portable Windows x64 build; I found that the same problems - no initial focus, and ctrl-F not working - beset that version too.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Shortcuts for focussing the searchbar are:
Alt + D or Command/Ctrl + L

Retrieved from https://docs.freetubeapp.io/usage/keyboard-shortcuts/

**Comment by LinuxOnTheDesktop** [comment]
You make me realise that my point is as follows. _If for some reason the search bar shall not be focused upon start-up_ (but _why not?_ or is it _that_ which you could not reproduce?), _then at least an **intuitive** keyboard shortcut should focus the bar_.

Also: I find that, on Windows: alt-D does not focus the bar; ctrl-L does not focus the bar; Command-(i.e. the Windows key)-L . . locks my computer. But, ah: alt-D and ctrl-L _do_ focus the bar in the _nightly_.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Shortcuts are working as intended. Therefore im closing this.

**Comment by LinuxOnTheDesktop** [comment]
Let me quote (after a fashion) something from the project's Github:

![image](https://user-images.githubusercontent.com/47056543/173245498-cf80b3d6-fdc3-4142-a088-f0378adfc11b.png)

Let me now quote myself, twice:

> 
> **Expected Behavior**
> 
> Ideally, upon starting the program the search bar would be focused. 
> 


>  If for some reason the search bar shall not be focused upon start-up (but why not? or is it _that_ which you could not reproduce?), then at least an intuitive keyboard shortcut should focus the bar.

**Comment by absidue** [comment]
Hi @LinuxOnTheDesktop

FreeTube starts on the subscriptions page, focusing the search bar automatically might be annoying for people that want to look at their subscriptions first.

The `ALT+D` and `CTRL/CMD+L` keyboard shortcuts are intuitive for anyone that has used keyboard shortcuts in a web browser, like in FreeTube those keyboard shortcuts focus the main search bar, also known as that location bar or address bar. When you search for something in the main search bar in a web browser or FreeTube the page changes, in web browsers the `CTRL+F` keyboard shortcut opens and focuses a search box that only searches on the current page. In FreeTube we have various search field that search on the current page, e.g. searching through history, playlist or channels. Currently no keyboard shortcut has been configured to focus those search boxes, however if one were to be implemented then that is the proper use for the `CTRL+F` keyboard shortcut.
